src/test/java/nearsoft/academy/bigdata/recommendation/MovieRecommenderTest.java

```diff
@@ -12,19 +12,24 @@
 
 public class MovieRecommenderTest {
     @Test
-    public void testDataInfo() throws IOException, TasteException {
-        //download movies.txt.gz from 
-        //    http://snap.stanford.edu/data/web-Movies.html
-        MovieRecommender recommender = new MovieRecommender("/path/to/movies.txt.gz");
-        assertEquals(7911684, recommender.getTotalReviews());
-        assertEquals(253059, recommender.getTotalProducts());
-        assertEquals(889176, recommender.getTotalUsers());
+    public static void testDataInfo() throws IOException, TasteException {
+        // download movies.txt.gz from
+        // http://snap.stanford.edu/data/web-Movies.html
+        MovieRecommender recommender = new MovieRecommender("/Users/csuarez/Downloads/movies.txt.gz");
```

Comment:
you are the only one with this path available
a placeholder is probably  good option, like the one that was before